exception_notification.gemspec

```diff
@@ -34,4 +34,5 @@ Gem::Specification.new do |s|
   s.add_development_dependency "slack-notifier", ">= 1.0.0"
   s.add_development_dependency "aws-sdk-sns", "~> 1"
   s.add_development_dependency "dogapi", ">= 1.23.0"
+  s.add_development_dependency "timecop", "~>0.8.0"
```

Comment:
Is there a reason for limiting timecop to version `< 0.9.0`? 

Latest version is `0.9.1`

Reply:
no, I think 0.9.1 should work. Updated here 61a8ef579b9f6c8ff01a8eb115851413bb6030a6